[docs] Add a note that ToggleButton exclusive does not enforce selection (#29812)
mui-org/material-ui: docs/src/pages/components/toggle-button/toggle-button.md

In this example, text justification toggle buttons present options for left, center, right, and fully justified text (disabled), with only one item available for selection at a time.

> **Note**: Exclusive selection does not enforce that at least one button must be active. For that effect see [enforce value set](#enforce-value-set).

Reviewer: I'm not sure about "at least one", since with exclusive it can only be one (or none).
```suggestion
**Note**: Exclusive selection does not enforce that a button must be active. For that effect see [enforce value set](#enforce-value-set).
```
